c2corg_ui/templates/outing/edit.html

```diff
@@ -563,14 +567,14 @@ <h5 translate ng-show="outing.associations.users.length == 0">You can look up fo
 
       </div>
 
+      ## FIXME: really useful?
       <div class="step step-4"></div>
```

Comment:
Do I have to update the maxSteps here https://github.com/c2corg/v6_ui/blob/ba802ec7f9112b241489d6d2837e8720fc6a7c11/c2corg_ui/templates/outing/edit.html#L36 ?